packages/paste-website/src/pages/components/visual-picker/index.mdx

```diff
@@ -0,0 +1,418 @@
+};
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/user-dialog"
+  storybookUrl="/?path=/story/components-user-dialog--"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+<LivePreview
+  scope={{
+    VisualPickerRadioGroup,
+    VisualPickerRadio,
+    useUID,
+    Text,
+    Box,
+    MediaObject,
+    MediaFigure,
+    MediaBody,
+    Avatar,
+    AvatarGroup,
+    StatusBadge,
+    CodeIcon,
+    CommunityIcon,
+  }}
+  noInline
+  showOverflow
+>
+  {VisualPickerRadioVertical}
+</LivePreview>
+
+## Guidelines
+
+### About Visual Picker
+
+Visual Picker is a selection component that allows users to select from a set of options. It is ideal to use in situations where users need to be able to see and compare different options before making a selection.
+
+This component is built on top of [Radio Group](/components/radio-group) and [Checkbox](/components/checkbox) guidelines.
+
+### Accessibility
+
+Visual Picker follows the same accessibility guidelines as [Radio Group](/components/radio-group) and [Checkbox](/components/checkbox).
+
+Like all input elements, both Radio and Checkbox Visual Pickers require a group label that describes the collection. Provide an accessible label using the `legend` prop on VisualPickerRadioGroup or VisualPickerCheckboxGroup. The grouping should be visually implicit, but a description still needs to be provided for assistive technology.
+
+Always use the `labelText` prop on VisualPickerRadio and VisualPickerCheckbox to provide a descriptive string for each option.
+
+When focused on a Visual Picker Radio control, use arrow keys to navigate to the other radios. When focused on a Visual Picker Checkbox control, use the tab key to navigate to the other checkboxes.
+
+## Examples
+
+### Visual Picker Radio
+
+Use a Visual Picker Radio when a user must select a single option.
+
+#### Vertical
+
+Vertical is the default orientation for Visual Picker.
+
+<LivePreview
+  scope={{
+    VisualPickerRadioGroup,
+    VisualPickerRadio,
+    useUID,
+    Text,
+    Box,
+    MediaObject,
+    MediaFigure,
+    MediaBody,
+    Avatar,
+    AvatarGroup,
+    StatusBadge,
+    CodeIcon,
+    CommunityIcon,
+  }}
+  noInline
+  showOverflow
+>
+  {VisualPickerRadioVertical}
+</LivePreview>
+
+#### Horizontal
+
+Use the horizontal Visual Picker when more complex visuals will help users compare information. Avoid using horizontal if there are more than 3-4 options.
+
+<LivePreview
+  scope={{
+    VisualPickerRadioGroup,
+    VisualPickerRadio,
+    useUID,
+    Text,
+    Box,
+  }}
+  noInline
+  showOverflow
+>
+  {VisualPickerRadioHorizontal}
+</LivePreview>
+
+### Visual Picker Checkbox
+
+Use a Visual Picker Checkbox when a user can select multiple options.
+
+#### Vertical
+
+Vertical is the default orientation for Visual Picker.
+
+<LivePreview
```

Comment:
Can we make the label "Select service" and help text "The message will be sent to all recipients for each selected service."